guides/source/active_job_basics.md

```diff
@@ -131,77 +118,331 @@ That's it!
 
 You can enqueue multiple jobs at once using [`perform_all_later`](https://api.rubyonrails.org/classes/ActiveJob.html#method-c-perform_all_later). For more details see [Bulk Enqueuing](#bulk-enqueuing).
 
-Job Execution
--------------
+Default Adapter: Solid Queue
```

Comment:
This is a new section for Solid Queue.

We use the term "adapter" and "backend" interchangeably in the documentation. I'm open to standardizing if we have a preference.

Reply:
Thnaks! Changed in some of the new content where it made sense.